### DISCUSSION TITLE
Linear Color Map

### DISCUSSION THREAD

**Discussion by shi093** [opengeos/leafmap#125] [Q&A] [answered]
I would like to create a color map which can be used to color polygons based on a variable value. Does leafmap colormap module has similar functions as this:

import branca.colormap as cm
colormap = cm.LinearColormap(colors= , index=, vmin= ,vmax=)

def fill_color(feature):
    return {
        "fillColor": colormap(feature['properties'][color_by])
    }

**Comment by giswqs** [comment, marked as answer]
You can the leafmap colormap moule `get_palette(palette, n_colors)` function to generate a linear color palette and then assign colors to your data. Similar to what I did in this web app https://streamlit.gishub.org. I will try to add a function to make it easier to use when I have time. 
Code: https://github.com/giswqs/streamlit-geospatial/blob/master/apps/housing.py#L336
